### PR TITLE
performance improvements

### PR DESCRIPTION
**Pull request by anha1**

### CODE REVIEW

File: amqp/src/main/resources/amqp.conf [hunk 1]
```diff
@@ -20,7 +20,7 @@ config.amqp = {
   consumerThreadPoolQueueCapacity = -1
 
   # Interval of the heartbeats that are used to detect broken connections. Zero for none. See for more details: https://www.rabbitmq.com/heartbeats.html
-  heartbeatIntervalSec = 1
+  heartbeatIntervalSec = 30
```
Comment (anha1) [inline amqp/src/main/resources/amqp.conf:23]: with this 1s delay msb-java loses connection under a heavy load

File: core/src/main/java/io/github/tcdl/msb/collector/CollectorManager.java [hunk 2]
```diff
@@ -53,26 +53,21 @@ public void registerCollector(Collector collector) {
...
      */
     public void unregisterCollector(Collector collector) {
         collectorsByCorrelationId.remove(collector.getRequestMessage().getCorrelationId());
-
-        synchronized (this) {
-            if (collectorsByCorrelationId.isEmpty() && isSubscribed) {
-                channelManager.unsubscribe(topic);
-                isSubscribed = false;
-            }
-        }
```
Comment (anha1) [inline core/src/main/java/io/github/tcdl/msb/collector/CollectorManager.java:75]: As far as responses topic is static, I don't see a need in this "unsubscripe" call that has a significant negative performance impact (a lot of subscribe/unsubscribe cycles). I'll create msb-node change request regarding it.

File: amqp/src/main/resources/amqp.conf (hunk 1 above)
Comment (alexandr-zolotov) [inline amqp/src/main/resources/amqp.conf:23]: Don't you know why?

Comment (anha1) [inline amqp/src/main/resources/amqp.conf:23]: The application can't deliver heartbeats each second under a heavy load, so RabbitMQ closes a connection on a missing heartbeat. AFAIR, we faced this issue before during our MS load testing. So I'm considering the previous 1s default value as dangerous and harmful. RabbitMQ defaults are also far from 1s: "The timeout is in seconds, and default value is 60 (580 prior to release 3.5.5).".

**Pull request merged**